cpp/src/fil/treelite_import.cu

```diff
@@ -200,16 +213,14 @@ cat_sets_owner allocate_cat_sets_owner(const tl::ModelImpl<T, L>& model)
   return cat_sets;
 }
 
-void adjust_threshold(
-  float* pthreshold, int* tl_left, int* tl_right, bool* default_left, tl::Operator comparison_op)
+void adjust_threshold(float* pthreshold, bool* swap_child_nodes, tl::Operator comparison_op)
```

Comment:
I can no longer swap indices since they're not passed by reference. It actually simplifies the code, since swapping and related adjustments are done only once.

Reply:
This is fine, and the new code looks cleaner.